$NetBSD: patch-ad,v 1.3 2017/02/24 00:10:05 maya Exp $

Don't build va_trace.c - it requires linux-specific stuff,
and is used for tracing (and doesn't seem necessary).

--- va/Makefile.am.orig	2016-11-10 00:34:35.000000000 +0000
+++ va/Makefile.am
@@ -33,7 +33,6 @@ libva_source_c = \
 	va.c			\
 	va_compat.c		\
 	va_fool.c		\
-	va_trace.c		\
 	$(NULL)
 
 libva_source_h = \
